indexer/src/main/java/org/uniprot/store/indexer/keyword/KeywordStatisticsReader.java

```diff
@@ -9,6 +9,10 @@
 
 /** @author lgonzales */
 public class KeywordStatisticsReader implements RowMapper<KeywordStatisticsReader.KeywordCount> {
+    public static final String KEYWORD_STATISTICS_URL =
```

Comment:
Can you please move this constant to KeywordStatisticsStep  that is the class that is using it.. This way you can change it to private.